requirements.txt

```diff
@@ -1,4 +1,4 @@
-DLite-Python>=0.3.3,<1.0
+DLite-Python>=0.4.3,<1.0
```

Comment:
This probably needs to increase to 0.4.4 once the bug fix from https://github.com/SINTEF/dlite/pull/682 is released

Reply:
Done